app.py

```diff
@@ -44,6 +44,7 @@
        "mode": None,                                   # What orchestation type? "rancher" or "docker"
        "reaper_timeout_secs": 600,                     # How long should a container be idle before it gets reaped?
        "reaper_ipnetwork": u"127.0.0.1/32",            # What IP address/network is allowed to access /reaper/ ?
+       "zombie_service_prefix": "narrative",           # Prefix for name of services zombie reaper should reap (use empty string to match any name)
```

Comment:
I used the existing cfg['container_name'] with a null parameter as the prefix match for reaping candidates: https://github.com/kbase/narrative-traefiker/blob/a2990d7c422bce97659f6253b7edf769c2fa52af/app.py#L360

If you want to match the prespawns as well you could add in a check for  cfg['container_name_prespawn'].format('') as well.

The concern is that if the values for cfg['container_name'] or cfg['container_name_prespawn'] get changed, then cfg['zombie_service_prefix'] becomes ineffective. I think its better to use the same configuration item used to name the containers in the first place.

Reply:
Ugh, I knew we had variables like that already!  Fixed.